stackslib/src/core/mod.rs

```diff
@@ -30,9 +30,12 @@ use crate::burnchains::bitcoin::BitcoinNetworkType;
 use crate::burnchains::{Burnchain, Error as burnchain_error};
 use crate::chainstate::burn::ConsensusHash;
 pub mod mempool;
+pub mod nonce_cache;
 
 #[cfg(test)]
 pub mod tests;
+#[cfg(any(test, feature = "testing"))]
+pub mod util;
```

Comment:
I'd prefer calling this module something like `test_util`

Reply:
Moved in d7df7c99417be4b00f85d1f27055f158886df96f